Sprint 4

wjarvis2 opened the pull request.

wjarvis2 committed:
Add files via upload

wjarvis2 committed:
Add files via upload

wjarvis2 committed:
Add files via upload

wjarvis2 committed:
Add files via upload

07sebascode commented:
Assignment Jan 28: Well done, nice way to replace the values on the dataframe, it is always useful to double check for null values like you did. Using a pivot table at the beginning gives a clear view of the data, and helps getting some idea of each party’s preference for each issue, and having separate dataframes for Republicans and Democrats made the next steps easy to solve and understand. What you did with code gives you all the answers you need for each of the questions on the assignment, just remember writing down with your own words the answer to each of the questions based on your results, like mentioning a specific issue that democrats support more than republicans with p < 0.01, mentioning a specific issue that republicans support more than democrats with p < 0.01, and mentioning a specific issue where the difference between republicans and democrats has p > 0.1

07sebascode commented:
Assignment Jan 29: Good job importing the data, nice way to replace the values on the dataframe, you are on the right track, but still have not accomplished the last 2 objectives:

-Graphically (with a plot) represent the confidence interval
-Interpret the confidence interval - what does it tell you about the data and its distribution?

wjarvis2 committed:
Add files via upload

wjarvis2 committed:
Add files via upload

I didn't finish this one. Everything was incredibly easy with the exception that I somehow couldn't figure out how to make the contingency table. I am looking forward to the opportunity to attempt this one again.

wjarvis2 committed:
Add files via upload
